backend/webapp/routes/offchain.py

```diff
@@ -184,6 +190,58 @@ def get(self):
                 HTTPStatus.OK,
             )
 
+    class AddPaymentCommand(OffchainView):
```

Comment:
This endpoint is not adding just any payment command, but only the commands where I am a sender. It should be reflected somehow in the name.